Library/OcBootManagementLib/BootAudio.c

```diff
@@ -194,6 +194,14 @@ OcPlayAudioEntry (
     OcPlayAudioFile (Context, OcVoiceOverAudioFileExternalOS, FALSE);
   } else if (Entry->Type == OC_BOOT_RESET_NVRAM) {
     OcPlayAudioFile (Context, OcVoiceOverAudioFileResetNVRAM, FALSE);
+  } else if (Entry->Type == OC_BOOT_TOGGLE_SIP) {
+    if (StrCmp (Entry->Name, OC_MENU_SIP_IS_DISABLED) == 0) {
+      OcPlayAudioFile (Context, OcVoiceOverAudioFileSIPIsDisabled, FALSE);
+    } else if (StrCmp (Entry->Name, OC_MENU_SIP_IS_ENABLED) == 0) {
+      OcPlayAudioFile (Context, OcVoiceOverAudioFileSIPIsEnabled, FALSE);
+    } else {
+      OcPlayAudioFile (Context, OcVoiceOverAudioFileExternalTool, FALSE);
```

Comment:
Is this possible? In my opinion, ASSERT.

Reply:
Yes